src/xhr/components/PostGetDeleteExample.md

```diff
@@ -1,6 +1,15 @@
-POST | GET | DELETE Examples
+Click the `Get Posts` button below to trigger a GET request, then hover an item and
+click its 'X' to trigger a DELETE a particular item. You can also use the
+_Create a Post_ form at the bottom to trigger a POST and create a new item:
 
-```jsx
+```jsx noeditor
```

Comment:
This example snippet will no longer be shown, but will result in the component rendering. The code fence with `js { "fille": ...` will, however, use the entire file so folks can see how we used the _Xhr_ components and hooks:

![image](https://user-images.githubusercontent.com/142403/69903145-c44b5380-134a-11ea-9de4-2e747089d6ba.png)